scipy/stats/tests/test_crosstab.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+from numpy.testing import assert_array_equal
+from scipy.stats import crosstab
+
+
+def test_crosstab_basic():
+    a = [0, 0, 9, 9, 0, 0, 9]
+    b = [1, 2, 3, 1, 2, 3, 3]
+    expected_avals = [0, 9]
+    expected_bvals = [1, 2, 3]
+    expected_count = np.array([[1, 2, 1],
+                               [1, 0, 2]])
+    (avals, bvals), count = crosstab(a, b)
+    assert_array_equal(avals, expected_avals)
+    assert_array_equal(bvals, expected_bvals)
+    assert_array_equal(count, expected_count)
```

Comment:
Worth adding test where input arguments do not occur in `np.unique` order?

Reply:
I just tweaked the order of the first two elements in the second argument of the first test.  Does that cover this case sufficiently?

Reply:
Yes, that's fine.